runtime/src/test/java/org/corfudb/runtime/view/ClusterHealthTest.java

```diff
@@ -0,0 +1,33 @@
+package org.corfudb.runtime.view;
+
+import org.corfudb.runtime.view.ClusterStatusReport.ClusterStatus;
+import org.corfudb.runtime.view.ManagementView.ClusterHealth;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ClusterHealthTest {
```

Comment:
Can you please explain what the test does? What is the expected behavior and why.

Reply:
done